src/Transfers/Constants.php

```diff
@@ -207,7 +209,7 @@ class Constants
     const REASON_NO_REACTION_BUYER = 'NoReactionBuyer';
     const CARRIER_CODE_OTHER = 'Other';
     const CARRIER_CODE_OTHER_HAULER = 'Other Hauler';
-    const CARRIER_CODE_AMAZON_LOGISTICS_DE_SWISHIP = 'Amazon Logistics DE (Swiship)';
+    const CARRIER_CODE_AMAZON_LOGISTICS_DE_SWISHIP_ = 'Amazon Logistics DE (Swiship)';
```

Comment:
NIT: Is there any reason, why we need this trailing dash?

```suggestion
    const CARRIER_CODE_AMAZON_LOGISTICS_DE_SWISHIP = 'Amazon Logistics DE (Swiship)';
```

Reply:
Probably not but this was auto-generated so I didn't change it.